.github/workflows/appium-automation.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         with:
           path: qa
           repository: BranchMetrics/qentelli-saas-sdk-testing-automation
+          ref: ReInstallationScenario
```

Comment:
Why are we referencing ReInstallationScenario here as this PR is only updating env variable?

Reply:
Yes @jyotisoma-branch 
you are right. Along with that we have added optimisation, create QR code feature implementation and parallel execution changes are there in the ReInstallationScenario branch. 
I have added all the details in the automation PR. Here is the link for automation PR link https://github.com/BranchMetrics/qentelli-saas-sdk-testing-automation/pull/43. Could you please check this and confirm once.
Thanks.